go.sum

```diff
@@ -1,336 +1,35 @@
-bazil.org/fuse v0.0.0-20160811212531-371fbbdaa898/go.mod h1:Xbm+BRKSBEpa4q4hTSxohYNQpsxXPbPry4JJWOB3LB8=
```

Comment:
Do we not need the `go.sum` file?

Reply:
File is still present. Most of the contents of the file have been removed.